sinks/multi_sink.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+import sys
+
+MAX_READLINES = 1000
+
+
+class Sinks:
+
+    def __init__(self, sink_commands):
+        self.sinks = []
+        self.lines_sent = 0
+        for param in sink_commands:
+            print(f'Executing: {param}')
+            self.sinks.append(subprocess.Popen(param.split(), stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.STDOUT))
+
+    def write_lines(self, lines):
+        self.lines_sent += len(lines)
+        encoded_lines = [line.encode('utf-8') for line in lines]
+        [sink.stdin.writelines(encoded_lines) for sink in self.sinks]
+
+
+sinks = Sinks(sys.argv[1:])
```

Comment:
Can you put an example of how it is supposed to be configured?

This is our current configuration: https://github.com/Telefonica/dockerfiles/blob/master/statsite/statsite.conf#L15

Reply:
I'm not sure if you ask to put the example here or in the code. I'll put it here and you tell me.
In an scenario where we want to send to different ports in the same server, we would use something like:

`python multi_sink.py "python /sinks/graphite.py GRAPHITE_HOST 2004 'stats.' 3 'pickle'" "python /sinks/graphite.py GRAPHITE_HOST 2005 'stats.' 3 'pickle'"`